db/schema.rb

```diff
@@ -10,8 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2022_11_24_155505) do
```

Comment:
FYI I'm aware of the extra `db/schema.rb` changes here - I'll get these reverted shortly